operational_analysis/methods/turbine_long_term_gross_energy.py

```diff
@@ -381,11 +381,13 @@ def filter_sum_impute_scada(self):
         self._scada_daily_valid.reset_index(inplace = True)
         
         # Impute missing days for each turbine - provides progress bar
-        self._scada_daily_valid['energy_imputed'] = imputing.impute_all_assets_by_correlation(self._scada_daily_valid, 
-                                                                                              input_col = 'energy_kwh_corr',
-                                                                                              ref_col = 'energy_kwh_corr',
-                                                                                              align_col = 'day',
-                                                                                              id_col = 'id')
+        self._scada_daily_valid['energy_imputed'] = imputing.impute_all_assets_by_correlation(
+            input_col=self._scada_daily_valid.energy_kwh_corr,
+            ref_col=self._scada_daily_valid.energy_kwh_corr,
+            align_col=self._scada_daily_valid.day,
+            id_col=self._scada_daily_valid.id,
+            index_col=self._scada_daily_valid.index.values,
+        )
```

Comment:
_scada_daily_valid is already a pandas dataframe.

Reply:
Yeah, that's my understanding as well, this was to get the functionality in line with what's expected for input types for the imputing function call.